docs/explanation/development-process.rst

```diff
@@ -1,17 +1,320 @@
+beginning of each cycle. The package build system is set up, the toolchain
+is organised, :term:`seeds` are branched, and many other pieces are made ready
+before development can properly begin. Once these preparations are made, the
+new branch is officially announced on the
+`ubuntu-devel-announce mailing list <https://lists.ubuntu.com/mailman/listinfo/ubuntu-devel-announce>`_ 
+and opened for uploads to the :doc:`/explanation/archive`.
+
+.. note::
+    See the `Ubuntu 24.04 LTS (Noble Numbat) archive opening announcement email <https://lists.ubuntu.com/archives/ubuntu-devel-announce/2023-October/001341.html>`_
+    as an example.
+
+Planning
+--------
+
+Ubuntu contributors discuss the targeted features for each release cycle via
+the various channels (e.g., IRC, Matrix, Discourse, Launchpad). Some of these
+come from strategic priorities for the distribution as a whole, and some are
+proposed by individual developers.
+
+The broader open-source community gets together at the :term:`Ubuntu Summit` 
+(similar but different to the past 
+:term:`Ubuntu Developer Summits <Ubuntu Developer Summit>`) to share
+experiences and ideas and to inspire future exciting projects covering
+development as well as design, writing, and community leadership with a wide
+range of technical skill levels.
+
+The :term:`Canonical` teams meet after every release at the Roadmap Planning 
+Sprint to plan the roadmap for the next release and discuss how to implement
+the roadmap at the Engineering Sprint.
+
+Merging with Upstream & feature development
+-------------------------------------------
+
+The first phase of the release cycle is characterised by bringing new releases
+of :term:`upstream` components into Ubuntu, either directly or via 
+:doc:`Merges and Syncs from Debian </explanation/debian-merges-and-syncs>`. 
+Development of planned projects for the release often begins while merging is
+still underway, and their development is accelerated once the package archive
+is reasonably consistent and usable.
+
+The automatic import of new package versions from Debian ends at the 
+:ref:`DebianImportFreeze`.
+
+Stabilisation and deadlines (freezes)
+-------------------------------------
+
+During development, caution is increasingly exercised in making changes
+to Ubuntu to ensure a stable state is reached in time for the final release
+date. The typical order and length of the various freezes can be seen on the
+current Release Schedule, which is usually posted as a Discourse article under
+the `"Release" topic <https://discourse.ubuntu.com/c/release/>`_. 
+
+.. note::
+    In the past, the Release Schedule was published in the Ubuntu Wiki.
+    See for example the `release schedule of Ubuntu 20.04 LTS (FocalFossa) <https://wiki.ubuntu.com/FocalFossa/ReleaseSchedule>`_.
+
+During freezes, exceptions must be requested to approve changes.
+See :doc:`how to request a freeze exception </how-to/request-freeze-exception>`.
```

Comment:
@s-makin what do you think about:

```suggestion
During development, caution is increasingly exercised in making changes
to Ubuntu to ensure a stable state is reached in time for the final release
date. Modifications to the Ubuntu package archive get incrementaly restricted,
effectively freezing the state of the Ubuntu package archive. The
deadlines when these restrictions get enabled are called "freezes". During
freezes, exceptions must be requested to approve changes. See 
:doc:`how to request a freeze exception </how-to/request-freeze-exception>`.
The typical order and length of the various freezes can be seen on the
current Release Schedule, which is usually posted as a Discourse article under
the `"Release" topic <https://discourse.ubuntu.com/c/release/>`_. 

.. note::
    In the past, the Release Schedule was published in the Ubuntu Wiki.
    See for example the `release schedule of Ubuntu 20.04 LTS (FocalFossa) <https://wiki.ubuntu.com/FocalFossa/ReleaseSchedule>`_.
```

Reply:
I think this is great, much clearer!

Reply:
Thanks, committed with https://github.com/canonical/ubuntu-packaging-guide/pull/62/commits/6425645543a860e8f087b99eeae60b1370c11540